kernel/src/constants.rs

```diff
@@ -1 +1,2 @@
-pub const VERSION: &str = "development";
+// pub const VERSION: &str = "development";
```

Comment:
This would break our release mechanism. See [comment here](https://github.com/DataDog/datadog-static-analyzer/pull/17#issuecomment-1627700809).

Reply:
Oh, I see. I didn't pay attention to that. I thought the idea was to use the kernel version in Cargo.toml but probably it doesn't make sense at this stage given that the crate is not public in crates.io or something like that. I'll revert that change.

Reply:
Yes, this is not really documented for now and we took the freedom to have our own system as this tool is published on GitHub release and not on crates.io. We are using the Git `sha` at build time so that we can clearly know what version the user is running. It helps a lot for troubleshooting. We have a similar system for the server, the `/version` endpoint returns the sha of the running code.